wiki/osu!_File_Formats/Db_(file_format)/fr.md

```diff
@@ -50,73 +50,74 @@ Quelques types de données sont spécifiques à osu!.db, les voici:
+| String | Nom du fichier audio |
+| String | Hash MD5 de la beatmap |
+| String | Nnom du fichier .osu correspondant à cette beatmap |
+| Byte | Statut de classement (4 = classée, 5 = approuvée, 2 = en attente/au cimetière) |
+| Short | Nombre de hitcircles |
+| Short | Nombre de sliders (note: cette valeur sera présnete pour tous les modes de jeu) |
+| Short | Nombre de spinners (note: cette valeur sera présnete pour tous les modes de jeu) |
+| Long | Dernière modification, ticks Windows. |
+| Byte/Single | Approach rate. Byte si la version est en-dessous de 20140609, Single sinon. |
+| Byte/Single | Circle size. Byte si la version est en-dessous de 20140609, Single sinon. |
+| Byte/Single | HP drain. Byte si la version est en-dessous de 20140609, Single sinon. |
+| Byte/Single | Overall difficulty. Byte si la version est en-dessous de 20140609, Single sinon. |
+| Double | Vélocité des sliders |
+| paire Int-Double* | Un Int indiquant le nombre de paires Int-Double suivantes, puis les paires susnommées. Évalutation en étoiles pour osu!standard, dans chaque paire, le Int est la combinaison des mods, et le Double l'évaluation. Seulement présent à partir de la version 20140609. |
+| paire Int-Double* | Un Int indiquant le nombre de paires Int-Double suivantes, puis les paires susnommées. Évalutation en étoiles pour osu!taiko, dans chaque paire, le Int est la combinaison des mods, et le Double l'évaluation. Seulement présent à partir de la version 20140609. |
+| paire Int-Double* | Un Int indiquant le nombre de paires Int-Double suivantes, puis les paires susnommées. Évalutation en étoiles pour osu!catch, dans chaque paire, le Int est la combinaison des mods, et le Double l'évaluation. Seulement présent à partir de la version 20140609. |
+| paire Int-Double* | Un Int indiquant le nombre de paires Int-Double suivantes, puis les paires susnommées. Évalutation en étoiles pour osu!manAia, dans chaque paire, le Int est la combinaison des mods, et le Double l'évaluation. Seulement présent à partir de la version 20140609. |
+| Int | Drain time, en secondes |
+| Int | Time total, en milliseconds |
+| Int | Le moment de la prévisualisation audio lorsque l'on survole une beatmap dans l'écran de sélection des musiques, en millisecondes. |
+| Timing point+ | Un Int indiquant le nombre de Timing points suivants, puis les Timing points indiqués. |
+| Int | Beatmap ID |
+| Int | Beatmap set ID |
+| Int | Thread ID |
+| Byte | Grade gagné en osu!standard. |
+| Byte | Grade gagné en osu!taiko. |
+| Byte | Grade gagné en osu!catch. |
+| Byte | Grade gagné en osu!mania. |
+| Short | Décalage local de la beatmap |
+| Single | Indulgence |
+| Byte | Osu gameplay mode. 0x00 = osu!standard, 0x01 = osu!taiko, 0x02 = osu!catch, 0x03 = osu!mania |
+| String | Source de la musique |
+| String | Tags de la musique |
+| Short | Décalage en ligne |
+| String | Police utilisée pour le titre de la musique |
+| Boolean | Est-ce que la beatmap est jouable |
```

Comment:
"Is Beatmap unplayed." Mets "True (vrai) si la beatmap n'a pas encore été jouée."